README.md

```diff
@@ -15,6 +15,10 @@
 ## What is Fluid?
 Fluid is an open source Kubernetes-native Distributed Dataset Orchestrator and Accelerator for data-intensive applications, such as big data and AI applications. It is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as a sandbox project.
 
+|<img src="https://user-images.githubusercontent.com/5708406/91202310-31eaab80-e734-11ea-84fc-c1b1882ae71c.png" height="24"/>&nbsp;Community Meeting|
```

Comment:
Is this image url a temporary place? I suggest moving the image into the current project?

Reply:
Hey @cheyang , this image url is not a temporary one. I noticed the CubeFS also used this one, check this https://github.com/cubeFS/cubefs/blob/master/README.md. CubeFS is also a CNCF sandbox project, so there is no IP issue, I think.